File: base-forge-security-spring-boot-starter/src/main/java/com/gewuyou/security/exception/AuthenticationException.java
package com.gewuyou.security.exception;

import com.gewuyou.core.exception.GlobalException;
import com.gewuyou.i18n.entity.ResponseInformation;

/**
 *认证异常
 *
 * @since 2024-11-27 23:59:29
 * @author gewuyou
 */
public class AuthenticationException extends GlobalException {
    public AuthenticationException(ResponseInformation responseInformation) {
        super(responseInformation);
    }
}

File: base-forge-security-authentication-spring-boot-starter/src/main/kotlin/com/gewuyou/baseforge/security/authentication/entities/exception/AuthenticationException.kt
package com.gewuyou.baseforge.security.authentication.entities.exception

import com.gewuyou.baseforge.autoconfigure.i18n.entity.ResponseInformation
import org.springframework.security.core.AuthenticationException as BaseAuthenticationException


/**
 * 认证异常
 *
 * @since 2024-11-27 23:59:29
 * @author gewuyou
 */
open class AuthenticationException(
    responseInformation: ResponseInformation
) : BaseAuthenticationException(
    responseInformation.responseI8nMessageCode
)
